src/types/handle.rs

```diff
@@ -202,7 +202,8 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES> + 'static> SystemContextHandl
     }
 
     /// begin consensus by sending a genesis event
-    pub async fn start_consensus(&self) {
+    #[deprecated]
+    pub async fn start_consensus_deprecated(&self) {
```

Comment:
Might be worth documenting what should be used instead

Reply:
This change is on me!  We're supposed to use start_consensus on SystemContext instead.  I can add a comment.